core/tests/src/test/java/org/jboss/jca/core/workmanager/unit/WorkManagerSecurityTestCase.java

```diff
@@ -178,6 +179,8 @@ public void testCallback() throws Throwable
     * @throws Throwable throwable exception
     */
    @Test
+   @Ignore
+   // test doesn't work because of lack of PicketBox integration which is no longer supported
```

Comment:
Instead of @Ignore can there be some kind of dynamic check in the test method that uses org.junit.Assume to throw AssumptionViolationException in cases where this shouldn't work or we don't care if it works (e.g. test is running on SE > 14 or where some EE API class's name starts with 'jakarta'?)

I ask because the poms in this PR indicate this will be coming in a micro release, so dropping tests in a branch used for micro releases is concerning.

Reply:
hi @bstansberry  I have removed all @ignores and have replace them wilt jakarta package check.

Reply:
@bstansberry thanks, merging